From eda2078837f00b8f929d1332ecf7565f88bedf41 Mon Sep 17 00:00:00 2001
From: Gil Forsyth <gil@forsyth.dev>
Date: Thu, 25 May 2023 10:41:24 -0400
Subject: [PATCH] set node version to 18.15.0

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ef5ed94a..55ad7c13 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,7 @@ requires = ["sphinx-theme-builder >= 0.2.0a10"]
 build-backend = "sphinx_theme_builder"
 
 [tool.sphinx-theme-builder]
-node-version = "18.16.0"
+node-version = "18.15.0"
 additional-compiled-static-assets = [
   "styles/furo-extensions.css",
 ]
-- 
2.34.1

